php/Dockerfile

```diff
@@ -34,12 +34,14 @@ ENV BUILDDEPS autoconf \
         libpspell-dev \
         libreadline-dev \
         libsasl2-dev \
+        libsodium-dev \
         libsqlite3-dev \
         libssh2-1-dev \
         libssl-dev \
         libsslcommon2-dev \
         libtidy-dev \
-        libxml2-dev \
+        libwebp-dev \
+	libxml2-dev \
```

Comment:
Indenting?

Reply:
Strange.  I don't see this spacing in VS code or when I look at the file in the github editor.  I'll see if I can figure out where/how it's being added.